ghedesigner/gfunction.py

```diff
@@ -253,7 +257,7 @@ def g_function_interpolation(self, b_over_h: float, kind="default"):
                 try:
                     d_values.append(self.D_values[h])
                 except Exception as e:  # noqa: BLE001
-                    print(e)
+                    logger.warn(e)
```

Comment:
One of these print statements is what was generating the new output you saw @mitchute. Previously there was no output when the `except` triggered. Now it should go only if it actually is a problem.

Reply:
Does that mean the exception is now being hit when it wasn't before?

Reply:
No, it means it was always being hit and was never giving any output/notification.

Reply:
Huh... why is it just now printing then?